pkg/cmd/update.go

```diff
@@ -0,0 +1,195 @@
+			if err != nil {
+				return err
+			}
+			version = release.TagName
+			changelog = release.Changelog
+		}
+		isCurrVerEqToPrev, err := isCurrentVersionEqualToPrevious(currentVersion, version)
+		if err != nil {
+			return err
+		}
+
+		if semver.Compare(currentVersion, version) >= 0 {
+			fmt.Println("Current version is greater than the version you are trying to update to")
+			return nil
+		}
+		if !isCurrVerEqToPrev {
+			changelog += "\n\nThere might be more important changes since you updated. Please visit https://github.com/daytonaio/daytona/releases for the complete changelog\n"
+		}
+		fmt.Println("Updating to version", version, "from", currentVersion)
+		fmt.Print("\nChangelog:\n\n")
+		fmt.Println(changelog)
+		return updateToVersion(version)
+
+	},
+}
+
+func init() {
+	updateCmd.Flags().StringVarP(&versionFlag, "version", "v", "", "Version to update to")
+
+}
+
+func fetchLatestRelase() (*GitHubRelease, error) {
+	resp, err := http.Get(githubReleaseURL + "/latest")
+	if err != nil {
+		return nil, fmt.Errorf("failed to fetch latest release: %w", err)
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusOK {
+		return nil, fmt.Errorf("failed to fetch latest release: %s", resp.Status)
+	}
+	var release GitHubRelease
+	if err := json.NewDecoder(resp.Body).Decode(&release); err != nil {
+		return nil, fmt.Errorf("failed to fetch latest release: %w", err)
+	}
+	return &release, nil
+}
+
+func fetchVersionRelease(version string) (*GitHubRelease, error) {
+	resp, err := http.Get(githubReleaseURL)
+	if err != nil {
+		return nil, fmt.Errorf("failed to fetch latest release: %w", err)
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusOK {
+		return nil, fmt.Errorf("failed to fetch latest release: %s", resp.Status)
+	}
+	var releases []GitHubRelease
+	if err := json.NewDecoder(resp.Body).Decode(&releases); err != nil {
+		return nil, fmt.Errorf("failed to fetch latest release: %w", err)
+	}
+	if len(releases) == 0 {
+		return nil, fmt.Errorf("no releases found")
+	}
+	for _, release := range releases {
+		if release.TagName == version {
+			return &release, nil
+		}
+	}
+	return nil, fmt.Errorf("version %s not found", version)
+}
+
+func updateToVersion(version string) error {
+	url, err := getBinaryUrl(version)
+	if err != nil {
+		return fmt.Errorf("failed to get binary url: %w", err)
+	}
+	resp, err := http.Get(url)
+	if err != nil {
+		return fmt.Errorf("failed to fetch binary: %w", err)
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusOK {
+		return fmt.Errorf("failed to fetch binary: %s", resp.Status)
+	}
+	err = update.Apply(resp.Body, update.Options{})
+	if err != nil {
+		fmt.Println("Failed to update binary")
+		if rollbackErr := update.RollbackError(err); rollbackErr != nil {
+			return fmt.Errorf("failed to rollback: %w", rollbackErr)
+		}
+		return fmt.Errorf("failed to update binary: %w", err)
+	}
+	fmt.Println("\nSuccessfully updated to version", version)
```

Comment:
Add `fmt.Println("If your Server is running, you need to restart it for the changes to take effect.")` so the user is aware that a server restart is required.